tests/eslint-rule-tester.js

```diff
@@ -8,12 +8,17 @@ const { RuleTester } = require("eslint")
 const { FlatRuleTester } = require("eslint/use-at-your-own-risk")
 const globals = require("globals")
 const semverSatisfies = require("semver/functions/satisfies")
+const os = require("os")
 
 // greater than or equal to ESLint v9
 exports.gteEslintV9 = semverSatisfies(eslintVersion, ">=9", {
     includePrerelease: true,
 })
 
+const platform = os.platform()
+exports.isCaseSensitiveFileSystem =
+    platform === "linux" || platform === "freebsd" || platform === "openbsd"
+
```

Comment:
Can we make this in a different file? I don't think it belongs with the rule tester

Reply:
right, renamed to `test-helpers`. :)